bin/node/src/cli/node.rs

```diff
@@ -1,47 +1,63 @@
 //! Node Subcommand.
 
-use crate::cli::globals::GlobalArgs;
+use crate::{cli::globals::GlobalArgs, node::RollupNode};
 use alloy_rpc_types_engine::JwtSecret;
 use anyhow::{Result, bail};
-use clap::Args;
+use clap::Parser;
 use kona_genesis::RollupConfig;
 use kona_registry::ROLLUP_CONFIGS;
 use serde_json::from_reader;
 use std::{fs::File, path::PathBuf};
 use url::Url;
 
 /// The Node subcommand.
-#[derive(Debug, Clone, Args)]
-#[non_exhaustive]
+///
+/// For compatibility with the [op-node], relevant flags retain an alias that matches that
+/// of the [op-node] CLI.
+///
+/// [op-node]: https://github.com/ethereum-optimism/optimism/blob/develop/op-node/flags/flags.go
+#[derive(Parser, Debug, Clone)]
+#[command(about = "Runs the consensus node")]
 pub struct NodeCommand {
+    /// URL of the L1 execution client RPC API.
+    #[clap(long, visible_alias = "l1", env = "L1_ETH_RPC")]
```

Comment:
nice if the rust docs here are also printed as help to cli, for this use clap attribute `verbatim_doc_comment`. seee use in 
https://github.com/paradigmxyz/reth/blob/18b71d9a90d95cc6e10a5c8b5839a6c9a8f71979/crates/cli/commands/src/p2p/mod.rs#L25